### PR TITLE
add initialized error

### PR DESCRIPTION
**Pull request by nolouch**
Ref https://github.com/pingcap/tidb/issues/23271
we need to handle the error in stale read.
Signed-off-by: nolouch <nolouch@gmail.com>

### CODE REVIEW

**Comment by NingLin-P** [comment]
LGTM

File: proto/errorpb.proto [hunk 1]
```diff
@@ -107,6 +114,7 @@ message ProposalInMergingMode {
     uint64 region_id = 1;
 }
 
+
```
Comment (JmPotato) [inline proto/errorpb.proto:117]:
```suggestion
```

**Comment by JmPotato** [comment]
@nolouch Should we merge this PR? I think the later error handling work will depend on this.

**Comment by nolouch** [comment]
PTAL @NingLin-P  @JmPotato

**Review by NingLin-P — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**